plugins/zenoh-plugin-storage-manager/src/replica/mod.rs

```diff
@@ -119,7 +119,7 @@ impl Replica {
 
         // Create channels for communication between components
         // channel to queue digests to be aligned
-        let (tx_digest, rx_digest) = flume::unbounded();
+        let (tx_digest, rx_digest) = flume::bounded(10);
```

Comment:
No, this is entirely a speculative first stab at an improvement. I certainly agree that a configurable value would be better than hardcoding anything. 

Truthfully, it feels like it would be better for this buffer management to be done entirely at the Zenoh level so that the backpressure happens when a Publisher tries to send its digest and it gets dropped because there's no space to receive. 

Even better, frankly, would be to remove this middle queue entirely and just have all the digest processing happen on recv from the digest-sub. I'm not entirely sure what the value of doing just the JSON parsing and `is_processed` checking separately from the rest of the handling is.